src/components/hero.js

```diff
@@ -75,6 +76,12 @@ const Arrow = ({ color, size }) => {
 const Banner = () => <ImgContainer data={banner}>Banner</ImgContainer>
 
 const Hero = ({ title, background }) => {
+  useEffect(() => {
+    if (process.env.NODE_ENV !== "development") {
```

Comment:
Not an issue yet, but I imagine this block of code would end up being repeated quite a bit. Maybe we incorporate this into the amplitude.js code so that if you log amplitude events in development, it'll just console.log, but if it's production it'll actually go to amplitude?

Reply:
Good idea. I will clean this up a bit.